js/lib/beautify.js

```diff
@@ -415,6 +415,7 @@
             return out;
         }
 
+        var _newline_restricted_tokens = /^(continue|break|return|throw)/;
```

Comment:
This should go with the other tokens and reserved word lists not buried in here.